docs/product/issues/issue-details/index.mdx

```diff
@@ -73,12 +73,18 @@ The trace navigator (which is displayed below the date) is an abbreviated view o
 A node will be red if it is associated with errors.
 Click "View Full Trace" to display the [Trace View](/product/sentry-basics/tracing/trace-view).
 
+{/* This feature requires multi-projects for the events endpoint */}
+<Include name="feature-available-for-plan-trial-team.mdx" />
```

Comment:
Can we move these notes up to right after their respective headings (line 61 & 80)? That way, users know whether it's relevant to them or not before reading the content